packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -198,28 +198,29 @@
 ///   }
 
 @mixin bpk-button--outline {
-  @include bpk-themeable-property(color, --bpk-button-outline-text-color, $bpk-color-white);
-  @include bpk-themeable-property(background-color, --bpk-button-outline-background-color, transparent);
+  background-color: $bpk-button-outline-background-color;
+  background-image: $bpk-button-outline-background-image;
+  color: $bpk-button-outline-color;
+
+  @include bpk-border-lg($bpk-button-outline-border-color);
```

Comment:
We should *always* define and use specific tokens if possible for mixins, as tokens are the lowest source of truth for the design system. So I've added a load of new ones e.g. `$bpk-button-outline-color`. I mostly copied the ones from the destructive button mixin.